partner_coc/data/res_partner_id_category_data.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
I keep it this way because when code gets changed the field coc_registration_number wont work properly.